addons/sale_management/views/sale_portal_templates.xml

```diff
@@ -32,28 +32,28 @@
                 <section>
                     <h3>Options</h3>
                     <t t-set="display_discount" t-value="True in [option.discount > 0 for option in sale_order.sale_order_option_ids]"/>
-                    <table class="table table-sm">
+                    <table class="table table-sm" name="sale_order_options_table">
                         <thead>
                             <tr>
-                                <th class="text-start">Product</th>
-                                <th t-if="display_discount" class="text-end">Disc.%
+                                <th class="text-start" name="th_product_name">Product</th>
+                                <th t-if="display_discount" class="text-end" name="th_discount">Disc.%
```

Comment:
Considering there is already a `th_discount` in the base template, wouldn't it be better to use an unique name like `th_option_discount` ?
Same remark for the `td` elements

Reply:
my vision was that you usually target a specific table then an element inside it, but indeed if existing xpath did not do that i might introduce uncertainty in the targeted xpath

corrected as suggested :)